### PR TITLE
Enables SRE for wasm

### PR DESCRIPTION
**Pull request by marek-safar**
Addressed also issues like #12225

### CODE REVIEW

File: mcs/class/System.Core/wasm_System.Core.dll.sources [hunk 1]
```diff
@@ -1 +1,15 @@
-#include testing_aot_full_System.Core.dll.sources
+#include common_System.Core.dll.sources
+#include dynamic_System.Core.dll.sources
+#include pipes_pns.sources
+
+System.Security.Cryptography/AesCryptoServiceProvider.cs
```
Comment (vargaz) [inline mcs/class/System.Core/wasm_System.Core.dll.sources]: Unrelated ?

Comment (marek-safar) [reply mcs/class/System.Core/wasm_System.Core.dll.sources]: no, it's part of NS

**Comment by marek-safar** [comment]
@monojenkins commit csproj
